### DISCUSSION TITLE
Klarna support ?

### DISCUSSION THREAD

**Discussion by pedrumiranda** [flutter-stripe/flutter_stripe#1134] [General]
Hello dear colleagues,

In the release notes I found someting about Klarna integration ?

Is this integration possible in the moment with Flutter_stripe ?

Thank you,
Pedor

**Comment by remonh87** [comment]
Hi yes this is possible: https://github.com/flutter-stripe/flutter_stripe/blob/main/example/lib/screens/regional_payment_methods/klarna_screen.dart

**Comment by pedrumiranda** [comment]
Hi Remon, thank you for the response. I'm just starting here with this package. In regards to the UI of Klarna integration, Does the UI also works out of the box with the PaymentSheet ? Or do we have to create something custom in regards to the UI ?
